### ISSUE TITLE
`tailwindcss-react-aria-components` missing `enabled:` modifier

### ISSUE THREAD

**Issue by somebody1234** [adobe/react-spectrum#6650]
### Provide a general summary of the feature here

[`enabled:`](https://tailwindcss.com/docs/hover-focus-and-other-states#enabled) is provided by tailwind as the opposite of `disabled:`. `tailwindcss-react-aria-components` overrides `disabled:` but not `enabled:`

### 🤔 Expected Behavior?

`enabled:` modifier works with RAC

### 😯 Current Behavior

`enabled:` modifier does not work with RAC

### 🔦 Context

I want to style a `Tab` differently only when it is not `disabled:`

**Comment by reidbarber** [comment]
Currently, all states in the plugin map to specific render props, and we don't have one for enabled. I think `[&:not([data-disabled])]:` should work for this case though. We could add an `enabled` to the plugin that maps to that if we are ok diverging from 1:1 mappings to render props though.
